AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.h

```diff
@@ -16,14 +18,33 @@ NS_ASSUME_NONNULL_BEGIN
 + (void)trackEvent:(NSString *)eventName;
 
 /**
- * Track an event.
+ * Track an event with properties.
  *
  * @param eventName  event name.
  * @param properties dictionary of properties.
  */
 + (void)trackEvent:(NSString *)eventName
     withProperties:(nullable NSDictionary<NSString *, NSString *> *)properties;
 
+/**
+ * Track an event with typed properties.
+ *
+ * @param eventName  event name.
+ * @param properties an MSEventProperties object.
+ *
+ * @discussion For events going to App Center, the following validation rules are applied:
+ *
```

Comment:
Extra spaces are not needed.

Reply:
They are there because when you look at the generated description, you only get line breaks this way. It's very confusing when it's all on one line (you can test by removing the extra lines and looking at the quick help generated).